src/css/layout/04-our-lessons.css

```diff
@@ -1,2 +1,210 @@
-.our__lessons {
+/* Our lessons Section */
+.lessons {
```

Comment:
You can remove this declaration: font-family: 'Manrope', sans-serif; since global styles already handle the font family.